<ui version="4.0" >
 <class>FormRoleRoleGorev</class>
 <widget class="QWidget" name="FormRoleRoleGorev" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Role ye Role Görevi Atama</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="ikon.qrc" >:/ikonlar/share.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Role Ara :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="ara_uyeler" />
   </item>
   <item row="0" column="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Role Görevi Ara:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QLineEdit" name="ara_kitaplar" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Roleler :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Tüm Görevler:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tablo_role" >
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QTableView" name="tablo_rolegorev" >
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Seçilen Role:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLineEdit" name="LeRole" />
   </item>
   <item row="3" column="3" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Seçilen Görev:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" >
    <widget class="QLineEdit" name="LeRoleGorev" />
   </item>
   <item row="4" column="0" colspan="5" >
    <widget class="QPushButton" name="PbAta" >
     <property name="text" >
      <string>Onayla</string>
     </property>
     <property name="icon" >
      <iconset resource="ikon.qrc" >:/ikonlar/onayla.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Sadece Bu Rolede ki Görevler:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5" >
    <widget class="QTableView" name="tablo_rolegorev_atanan" >
     <property name="mouseTracking" >
      <bool>true</bool>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="tabKeyNavigation" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSil" >
   <property name="icon" >
    <iconset resource="ikon.qrc" >:/ikonlar/eraser.png</iconset>
   </property>
   <property name="text" >
    <string>sil</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ikon.qrc" />
 </resources>
 <connections/>
</ui>
